packages/block-library/src/cover/edit.js

```diff
@@ -299,7 +298,6 @@ class CoverEdit extends Component {
 					<div className="wp-block-cover__inner-container">
 						<InnerBlocks
 							template={ INNER_BLOCKS_TEMPLATE }
-							allowedBlocks={ INNER_BLOCKS_ALLOWED_BLOCKS }
```

Comment:
This totally removes all the restrictions, e.g: it will be possible to add another cover or columns inside the cover block. @mtias is that the desired behavior or we should still have some restrictions in place but allow much more blocks than the ones we currently allow?

Reply:
I guess if this is the case then an exclude feature should be placed

Reply:
We could leave the default allowed blocks unchanged by adding a filter.
I guess there are many situations where you to exclude individual blocks.

Maybe this can be accomplished with filters.

```
// Option 1
const INNER_BLOCKS_ALLOWED_BLOCKS = applyFilters( 'blockLibrary.cover.overrideAllowedInnerBlocks', [ 'core/button', 'core/heading', 'core/paragraph' ] );

// Option 2, exclude the cover block
const innerBlocksAllowedBlocksFilter = applyFilters( 'blockLibrary.cover.overrideAllowedInnerBlocks', [ 'core/button', 'core/heading', 'core/paragraph' ] );
const INNER_BLOCKS_ALLOWED_BLOCKS = innerBlocksAllowedBlocksFilter.filter( block ) => ( block !== 'core/cover');

// Use addFilter to override the default
addFilter('blockLibrary.cover.overrideAllowedInnerBlocks', 'cover-allowed-innerblocks-hook', [ 'core/paragraph' ] );

```

Reply:
I thinking is that we either keep the allowed or never disallow anything, giving full freedom to the user if he wishes to nest covers, this is obviously debatable so I'm still up to any possible suggestions

Reply:
Let's start by removing the disallowed entirely. There can be use cases for nesting the same block for special visual effects.